.github/workflows/build.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: Set up NodeJS
         uses: actions/setup-node@v2
         with:
-          node-version: '12'
+          node-version: '13'
```

Comment:
I have upgraded to node 16 LTS on the server.

Reply:
Thanks. I'll adjust this in both Dockerfile and Github Actions.